pages/myplants.js

```diff
@@ -1,7 +1,8 @@
 import styled from "styled-components";
-import useSWR from "swr";
+import useSWR, { mutate } from "swr";
```

Comment:
```suggestion
import useSWR from "swr";
```
remove mutate as you are not using it